CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing to the CSH Constitution
+
+## Requirements
+Must have make, and pdflatex installed
+
+## Step by Step
+
+1. Fork this repo
+2. Make a new branch on your fork
+3. Modify the `.tex` files with your changes
+4. Run make *TWICE*
```

Comment:
Shouldn't have to run make twice anymore, the current makefile should compile twice so the links work. If that's not the case then we should make it the case.